### PR TITLE
Add default text in System Images when no image is present in Admin Panel

### LINKED ISSUE #2690: Fix Bug in System Images in Admin
**Describe the bug**

Currently a black page appears if there are no system images.
**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'Admin'
2. Click on 'Content'
3. Scroll down to 'System Images'

**Expected behaviour**

A text saying "No system images available" should be displayed.
**Screenshots**

Screenshot 2019-04-16 at 8 14 04 PM

**Additional context**

I am working on it.

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Fix Bug in System Images in Admin. Before a blank page appears if a new event topic was created without event subtopic as system images are only for subtopics. Now a text "No records to show" appears.

#### Changes proposed in this pull request:
Screenshot 2019-04-25 at 2 45 53 AM

Fixes #2690

### CODE REVIEW

File: app/templates/admin/content/system-images/list.hbs [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+  {{#if data}}
+    {{#each data as |subTopic|}}
+      <div class="eight wide column">
+        <h4>{{subTopic.name}}</h4>
+        {{widgets/safe-image isAvatar=true class='ui big image' src=(if subTopic.placeholder.originalImageUrl subTopic.placeholder.originalImageUrl
+          '/images/placeholders/avatar.png')}}
+        <div class="ui hidden divider"></div>
+        <button class="ui button primary" {{action 'openModal' subTopic.placeholder}} id="changebutton">{{t 'Change'}}</button>
+      </div>
+    {{/each}}
```
Comment (ritikamotwani) [inline app/templates/admin/content/system-images/list.hbs]: Use each else syntax.
https://guides.emberjs.com/v1.13.0/templates/displaying-a-list-of-items/#toc_empty-lists

File: app/templates/admin/content/system-images/list.hbs [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+  {{else}}
+    <div style="font-size:24px; padding:10px;">No system Images</div>
```
Comment (ritikamotwani) [inline app/templates/admin/content/system-images/list.hbs]: Check how helper text is added in rest of the places.

**Comment by harsh-jindal** [comment]
@ritikamotwani Made the changes, please review.

File: app/templates/admin/content/system-images/list.hbs [hunk 3]
```diff
@@ -7,6 +7,10 @@
       <div class="ui hidden divider"></div>
       <button class="ui button primary" {{action 'openModal' subTopic.placeholder}} id="changebutton">{{t 'Change'}}</button>
     </div>
+  {{else}}
+    <div class="ui basic segment">
```
Comment (ritikamotwani) [inline app/templates/admin/content/system-images/list.hbs]: No, this is not how helper text is added in the rest of the tables of the app.

**Comment by harsh-jindal** [comment]
@CosmicCoder96 I have changed the description and the commit message. Will keep the above point in mind from next time onwards.
@ritikamotwani I have implemented this the same as in #2517. 
Please review

**Comment by ritikamotwani** [comment]
@harsh-jindal , in the PR you've mentioned the helper text had to be like the other tables in that page itself.
For the rest of the tables "disabled header" class is used.
Check how it's done in the other places.

**Comment by CosmicCoder96** [comment]
@harsh-jindal Update this.

**Comment by harsh-jindal** [comment]
@CosmicCoder96 @ritikamotwani Made the changes. Please review.

**Pull request merged**